tools/verify_headers.py

```diff
@@ -15,7 +15,9 @@
 import multiprocessing
 import os
 import sys
+import re
 
+pep3120 = re.compile("^[ \t\f]*#.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)")
```

Comment:
Why did you use a regex here. We can just do a check for the string: `"# -*- coding: utf-8 -*-"`  or if you're worried about someone setting an non-utf8 coding string just check `line.startswith('# -*-coding)`

Reply:
I thought that this `#     -*-coding` was also something to avoid... What do you think about `'coding:' in string` ?

Reply:
Ah ok, I understand this a bit better now. You got this regex from PEP 263: https://www.python.org/dev/peps/pep-0263/#defining-the-encoding. In that case can you leave a comment on the source of this above.

```suggestion
# regex for character encoding from PEP 263
pep263 = re.compile(r"^[ \t\f]*#.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)")
```